src/nectarchain/dqm/camera_monitoring.py

```diff
@@ -4,11 +4,11 @@
 import numpy as np
 from astropy import time as astropytime
 from ctapipe.coordinates import EngineeringCameraFrame
-from ctapipe.instrument import CameraGeometry
 from ctapipe.visualization import CameraDisplay
-from dqm_summary_processor import DQMSummary
 from matplotlib import pyplot as plt
 
+from nectarchain.dqm.dqm_summary_processor import DQMSummary
+
```

Comment:
Covered in https://github.com/cta-observatory/nectarchain/pull/148/files#diff-593f6bdf396b1884f5f268dc53b094157f91cbb60ad009414793c0a40c143cde